synthtool/languages/node_mono_repo.py

```diff
@@ -460,19 +475,43 @@ def owlbot_main(
 
 
 def owlbot_entrypoint(
+    specified_owlbot_dirs: Optional[List[str]] = None,
     template_path: Optional[Path] = None,
     staging_excludes: Optional[List[str]] = None,
     templates_excludes: Optional[List[str]] = None,
     patch_staging: Callable[[Path], None] = _noop,
 ):
-    owlbot_dirs = walk_through_owlbot_dirs(Path.cwd())
-    for dir in owlbot_dirs:
-        owlbot_main(
-            dir, template_path, staging_excludes, templates_excludes, patch_staging
+    if specified_owlbot_dirs and specified_owlbot_dirs[0] == "all":
```

Comment:
nit: what if an empty list represented `all`, I think with this approach you could never have a directory called `all`?

Reply:
I considered that, but I think we want the default behavior to only run for changed files. In other words, normally, owlbot will just call the post processor without any args, like here: https://github.com/googleapis/repo-automation-bots/blob/32e02e7d8b497bb0a0a684c89615efab6ace75c8/packages/owl-bot/cloud-build/update-pr.yaml#L58

In that case, we only want it to check for changed files. So the reason I added an "all" argument was in case someone, for some reason, wanted to run it on all the subdirectories from the command line, i.e.:

```
docker run ${IMAGE} all
```

Reply:
I think I'd opt for `--all` rather than `all` (so that you don't have a weird edge case around folders named `all`). If this is annoying, since you now need to support flags, vs., just positional arguments, why don't we just add a comment:

```
# TODO: add support for running against all folders.
```